multiqc/modules/biscuit/biscuit.py

```diff
@@ -0,0 +1,688 @@
+            data['botGC-q40'] = (float(m2.group(1)) / float(m1.group(1)) * 100) if (m1 is not None and m2 is not None and float(m1.group(1))>0) else 0.0
+                
+        elif re.search(r'_covdist.*_table.txt', fn):
+
+            ## handles the following tables:
+            ## _covdist_q40_table.txt, _covdist_q40_botgc_table.txt
+            ## _covdist_q40_topgc_table.txt, _covdist_table.txt
+            ## _covdist_cpg_q40_table.txt, _covdist_cpg_q40_botgc_table.txt
+            ## _covdist_cpg_q40_topgc_table.txt, _covdist_cpg_table.txt
+            dd = {}
+            for l in f.splitlines():
+                fields = l.split()
+                dd[int(float(fields[0]))] = int(float(fields[1]))
+
+            covs = sorted([k for k in dd])[:30]
+            ccov_cnts = []
+            _ccov_cnt = sum(dd.values())
+            for cov in covs:
+                ccov_cnts.append(_ccov_cnt/1000000.)
+                _ccov_cnt -= dd[cov]
+
+            data = dict(zip(covs,ccov_cnts))
+        else:
+            raise Exception("Unknown file received: %s" % fn)
+                
+        return data
+
+    def chart_biscuit_coverage(self):
+
+        # base coverage
+        basecov = OrderedDict()
+        mdata = [
+            dict([(k.replace('_covdist_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_table.txt')]),
+            dict([(k.replace('_covdist_q40_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_q40_table.txt')]),
+            dict([(k.replace('_covdist_q40_botgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_q40_botgc_table.txt')]),
+            dict([(k.replace('_covdist_q40_topgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_q40_topgc_table.txt')]),
+            dict([(k.replace('_covdist_cpg_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_table.txt')]),
+            dict([(k.replace('_covdist_cpg_q40_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_q40_table.txt')]),
+            dict([(k.replace('_covdist_cpg_q40_botgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_q40_botgc_table.txt')]),
+            dict([(k.replace('_covdist_cpg_q40_topgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_q40_topgc_table.txt')]),
+        ]
+        if len(mdata)>0:
+            self.add_section(
+                name = 'Cumulative Base Coverage',
+                anchor = 'biscuit-coverage-base',
+                description = "<p>This plot shows the cummulative base coverage. High and low GC content region are the top and bottom 10% 100bp window in GC content.</p>",
+                plot = linegraph.plot(mdata, {'id':'biscuit_coverage_base',
+                    'title': 'Cumulative Base Coverage',
```

Comment:
Please add an X axis, `xLabelFormat` with the `X` suffix to denote coverage,  prepend the title with the module name.

Reply:
X-axis and xLabelFormat are added as suggested.